compiler/rustc_session/src/config.rs

```diff
@@ -1049,6 +1050,10 @@ impl OutputFilenames {
             .unwrap_or_else(|| OutFileName::Real(self.output_path(flavor)))
     }
 
+    pub fn interface_path(&self) -> PathBuf {
+        self.out_directory.join(format!("lib{}.{}", self.crate_stem, RUST_EXT))
```

Comment:
```suggestion
        self.out_directory.join(format!("lib{}.rs", self.crate_stem))
```
The constant isn't used anywhere else.